plugins/database/src/low_level_odbc.cpp

```diff
@@ -456,7 +460,8 @@ _cllExecSqlNoResult(
     icatSessionStruct* icss,
     const char*        sql,
     int                option ) {
-    rodsLog( LOG_DEBUG10, "%s", sql );
+
+    log_sql::debug("{}", sql);
```

Comment:
Include the function name in all log statements like so. It helps us see where the message comes from.
```cpp
log_sql::debug("{}: sql = [{}]", __func__, sql);
//              ^^^^             ^^^^^^^^
```

_This applies to all log-based changes in this PR._